ui/src/app/components/elements/elements-redirect.component.ts

```diff
@@ -29,9 +29,12 @@ export class ElementsRedirectComponent extends BaseComponent implements OnInit {
   ngOnInit() {
     this.route.parent.parent.parent.params.subscribe((params: Params) => {
       this.pushToParent(this.route, params);
-      if (this.router.url.endsWith("/elements")) {
-        this.elementFilterService.findAll(params.versionId).subscribe(res => {
-          this.navigation.replaceUrl(['/td', params.versionId, 'elements', 'filter', res.content.find(filter => filter.isDefault).id])
+      if (this.router.url.endsWith("/elements") || this.router.url.endsWith("/elements/create")) {
```

Comment:
paths can be maintained in the enums, I think, we already have one, plz have a check

Reply:
paths are not present in enums as of now, and this changes was done after verifying with jayavel bro for this elements-redirect file